bodhi/tests/server/test_bugs.py

```diff
@@ -300,7 +313,7 @@ def test_modified_exception(self, exception_log):
         bz._bz = mock.MagicMock()
         bz._bz.getbug.side_effect = Exception()
 
-        bz.modified(1411188)
+        bz.modified(1411188, 'A mean message.')
 
         bz._bz.getbug.assert_called_once_with(1411188)
```

Comment:
I guess we could add an assert to check the comment here

Reply:
We are adding an ```Exception``` side effect to the ```getbug``` method, and hence the remaining part will not be executed. Control will go to catch block and assertion of comment will not be possible.

Reply:
Make sense, I copied paste these comments a bit too fast :speak_no_evil: